<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>671</width>
     <height>261</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>669</width>
      <height>259</height>
     </rect>
    </property>
    <widget class="QLabel" name="BookName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>660</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookAuthor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>660</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookType">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>660</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookRate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>660</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="BookDes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>660</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Describe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="IDBook">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>660</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>IDBook:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txtSearch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>491</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WatchBook">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
